web/controllers/auth.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Notox.Auth do
+  import Plug.Conn
+
+  def init(_opts) do
+  end
+
+  defp get_current_user(conn) do
+    current_user_id = get_session(conn, :user_id)
+    current_user_id && Notox.Repo.get(Notox.User, current_user_id)
+  end
+
+  def call(conn, _opts) do
+    current_user = get_current_user(conn)
+    assign(conn, :current_user, current_user)
+  end
+
+  def login(conn, user) do
```

Comment:
This functionality doesn't really belong in the plug. I'd probably pull it out into another module. Maybe `Notox.Session`?